sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/EventProcessorBase.cs

```diff
@@ -390,29 +394,65 @@ private async Task RunAsync(CancellationToken cancellationToken)
                 // From the storage service provided by the user, obtain a complete list of ownership, including expired ones.  We may still need
                 // their eTags to claim orphan partitions.
 
-                var completeOwnershipList = (await ListOwnershipAsync(FullyQualifiedNamespace, EventHubName, ConsumerGroup)
-                    .ConfigureAwait(false))
-                    .ToList();
+                var completeOwnershipList = await ListOwnershipAsync(FullyQualifiedNamespace, EventHubName, ConsumerGroup)
```

Comment:
I think the majority of these changes are still largely relevant, even in the face of the major churn over the last few weeks.

@kinelski:  Any specific thoughts, given you're more familiar with the design in this area?